src/template.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To Do List</title>
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+</head>
+<body>
+  <div class="todo-container">
+    <div class="todo-header">
+      <h2 class="title">To-Do</h2>
+      <span class="refresh material-icons">refresh</span>
+    </div>
+    <hr>
+    <form action="#" method="post" name="taskForm" class="add-task-container">
+      <input class="add-task-input" type="text" name="description" id="addTask" placeholder="Add to your list..." required>
+      <span class="add-task-btn material-icons">keyboard_return</span>
+    </form>
+    <hr>
+    <div class="main-tasks"></div>
+    <div class="clear">
+      <button type="button" class="clear-btn">Clear all completed</button>
+    </div>
+  </div>
+</body>
```

Comment:
**[OPTIONAL]** Please follow the HTML semantics and wrap the body content inside the `main` tag, Wrapping your body content inside the `main` tag helps to clearly identify the most important part of your page, making it easier for search engines, accessibility tools, and other software to understand the structure of your content.